client/components/seo/preview-upgrade-nudge/index.jsx

```diff
@@ -47,10 +48,10 @@ export const SeoPreviewNudge = ( {
 				showIcon
 				plan={
 					site &&
-					findFirstSimilarPlanKey( site.plan.product_slug, {
-						type: TYPE_BUSINESS,
-						...( isJetpack ? { term: TERM_ANNUALLY } : {} ),
-					} )
+					findFirstSimilarPlanKey(
+						site.plan.product_slug,
+						isJetpack ? { type: TYPE_SECURITY_DAILY, term: TERM_ANNUALLY } : { type: TYPE_BUSINESS }
+					)
```

Comment:
I've left the Jetpack logic in here for now, but we shouldn't be loading this component for Jetpack sites.